initial_conditions:
  agent1:
    x: 0.0
    y: -2.0
    z: 0.5
    yaw: 1.57
  agent2:
    x: 1.0
    y: -2.0
    z: 0.5
    yaw: 3.14
  agent3:
    x: -1.0
    y: -2.0
    z: 0.5
    yaw: 0.0
  agent4:
    x: 2.0
    y: -1.0
    z: 0.5
    yaw: -1.57
  agent5:
    x: -2.0
    y: -3.0
    z: 0.5
    yaw: 1.57
  agent6:
    x: -1.0
    y: -3.0
    z: 0.5
    yaw: 1.57
  # agent7:
  #   x: 1.0
  #   y: -3.0
  #   z: 0.5
  #   yaw: 1.57


# initial_conditions:
#   agent1:
#     x: 0.0
#     y: -1.0
#     z: 0.5
#     yaw: 1.57
#   agent2:
#     x: 1.0
#     y: -1.0
#     z: 0.5
#     yaw: 1.57
#   agent3:
#     x: -1.0
#     y: -1.0
#     z: 0.5
#     yaw: 1.57
#   agent4:
#     x: 1.2
#     y: 0.0
#     z: 0.5
#     yaw: 1.6
#   agent5:
#     x: -1.0
#     y: 1.0
#     z: 0.5
#     yaw: 0.787
#   agent6:
#     x: 0.4
#     y: -2.1
#     z: 0.5
#     yaw: 1.75